app/scripts/BarTrack.js

```diff
@@ -432,15 +448,25 @@ class BarTrack extends HorizontalLine1DPixiTrack {
     this.visibleAndFetchedTiles()
       .filter((tile) => tile.svgData?.barXValues)
       .forEach((tile) => {
-        // const [xScale, xPos] = this.getXScaleAndOffset(tile.drawnAtScale);
         const data = tile.svgData;
 
         for (let i = 0; i < data.barXValues.length; i++) {
+          // hex colors with transparency cannot be directly used in SVG opened in Adobe Illustrator
+          // so alpha values are generated from the last two characters of the color string
+          const barColor = data.barColors[i];
+          const barColorFull =
+            barColor.length === 9
```

Comment:
What's the significance of `.length == 9`?

Would be good to add a comment explaining.

Reply:
I added a more complete explanation of why the color and alpha transparency hex values are split out to separate values.